dev-qt/qtwebengine-dicts/qtwebengine-dicts-1.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Hunspell dictionaries for QtWebEngine"
+HOMEPAGE="https://doc.qt.io/qt-5/qtwebengine-webenginewidgets-spellchecker-example.html"
+SRC_URI=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE=""
+
+S="${WORKDIR}"
+
+# Cribbed from app-text/hunspell
+LANGS="af bg ca cs cy da de de-1901 el en eo es et fo fr ga gl he hr hu ia id
+is it kk km ku lt lv mi mk ms nb nl nn pl pt pt-BR ro ru sk sl sq sv sw tn uk
+zu"
+
+DEPEND="
+	app-dicts/myspell-en
+	dev-qt/qtwebengine"
+
+for lang in ${LANGS}; do
+	IUSE+=" l10n_${lang}"
+	case ${lang} in
+		de-1901)	dict="de_1901";;
+		pt-BR)		dict="pt-br";;
+		*)			dict="${lang}";;
+	esac
+	DEPEND+=" l10n_${lang}? ( app-dicts/myspell-${dict} )"
+done
+unset dict lang
+
+RDEPEND="
+	${DEPEND}
+"
+
+src_configure() {
+	:
+}
+
+src_compile() {
+	local dicts
+	local dic
+	local bdic_fn
+	local prefix
```

Comment:
you can declare multiple vars in one line with `local`

Reply:
Sure, but I find that makes things less readable.  A number of style guides recommend against it in various languages.  Still, I'll change them if you insist.

Reply:
No, I'm fine either way.